packages/material-ui/src/PasswordWidget/PasswordWidget.tsx

```diff
@@ -1,49 +1,11 @@
 import React from "react";
 
-import TextField from "@material-ui/core/TextField";
+import { TextWidgetProps } from "../TextWidget";
 
-import { WidgetProps } from "@rjsf/core";
-
-const PasswordWidget = ({
-  id,
-  required,
-  readonly,
-  disabled,
-  value,
-  label,
-  onFocus,
-  onBlur,
-  onChange,
-  options,
-  autofocus,
-  schema,
-  rawErrors = [],
-}: WidgetProps) => {
-  const _onChange = ({
-    target: { value },
-  }: React.ChangeEvent<HTMLInputElement>) =>
-    onChange(value === "" ? options.emptyValue : value);
-  const _onBlur = ({ target: { value } }: React.FocusEvent<HTMLInputElement>) =>
-    onBlur(id, value);
-  const _onFocus = ({
-    target: { value },
-  }: React.FocusEvent<HTMLInputElement>) => onFocus(id, value);
-
-  return (
-    <TextField
-      id={id}
-      label={label || schema.title}
-      autoFocus={autofocus}
-      required={required}
-      disabled={disabled || readonly}
-      type="password"
-      value={value ? value : ""}
-      error={rawErrors.length > 0}
-      onFocus={_onFocus}
-      onBlur={_onBlur}
-      onChange={_onChange}
-    />
-  );
+const PasswordWidget = (props: TextWidgetProps) => {
+  const { registry } = props;
+  const { TextWidget } = registry.widgets;
+  return <TextWidget type="password" {...props} />;
```

Comment:
Will this render the exact same content as the old PasswordWidget component? Trying to see if this will be a breaking change or not.

Reply:
The snapshot update was to add the registry to the outer div (which seems weird and probably can be removed easily in my PR). The only other thing was adding an empty `placeholder` to the `input`, which honestly might have been a bug with the `PasswordWidget` before.

Reply:
@epicfaace let me know if stripping the `registry` from the outer div (as seen in the snapshots) is considered a breaking change.

Reply:
Yeah, I think it should be fine, not considered a breaking change since that doesn't affect how users use the library / any of the look-and-feel at all.